src/smc-webapp/projects.cjsx

```diff
@@ -513,7 +513,7 @@ class ProjectsStore extends Store
             return
         account_id = webapp_client.account_id
         list = []
-        if current? and map.has(current)
+        if current? and map.has(current)   #current is for current project
```

Comment:
Our style for comments is to put a space after the hash, so `# current is for current project`.  But might be to be more descriptive, e.g., `# current is the project_id of the current project`.